docs/admin/access.rst

```diff
@@ -9,62 +9,52 @@ Access control
     specifically built for Weblate. If you are using anything older, please consult
     the documentation for the specific version you are using.
 
-Weblate comes with a fine-grained privilege system to assign user permissions
-for the whole instance, or in a limited scope.
+After installing Weblate, you can grant permissions to various things on the
+whole installation using :ref:`default-groups`. :ref:`acl` lets you grant
+permissions to specific translation projects. If you want to fine-tune the
+permissions you give based on groups you make yourself, you can do so
+using :ref:`custom-acl`.
 
-The permission system is based on groups and roles, where roles define a set of
-permissions, and groups assign them to users and translations, see
-:ref:`auth-model` for more details.
-
-After installation a default set of groups are created, and you can use those
-to assign users roles for the whole instance (see :ref:`default-groups`). Additionally when
-:ref:`acl` is turned on, you can assign users to specific translation projects.
-More fine-grained configuration can be achieved using :ref:`custom-acl`.
-
-Common setups
--------------
-
-Locking down Weblate
```

Comment:
Moved this all the way to the bottom.
I don't see how locking down everything is common setup. There is rarely malice, and most instances don't run this. "Common setup → total lockdown" is a strange message to send IMO.